tests/test_exceptions.py

```diff
@@ -1,75 +1,119 @@
+    UnsupportedMethod,
+    UpgradeRequired,
+    VariantAlsoNegotiates,
 )
 from lbz.response import Response
+import pytest
 
 
 def test_lambda_fw_exception():
-    exp = LambdaFWException("Nope")
+    exp = LambdaFWException(message="Nope")
     assert exp.message == "Nope"
     assert exp.status_code == 500
     assert isinstance(exp.get_response(request_id=""), Response)
 
 
-def test_access_denied():
-    exp = AccessDenied()
-    assert exp.message == "Request forbidden -- authorization will not help"
-    assert exp.status_code == 403
-
-
-def test_bad_request_error():
-    exp = BadRequestError()
-    assert exp.message == "Bad request syntax or unsupported method"
-    assert exp.status_code == 400
-
-
-def test_invalid_resolution_error():
-    exp = InvalidResolutionError()
-    assert exp.message == "Unaccepted image resolution"
-    assert exp.status_code == 400
-
-
-def test_server_error():
-    exp = ServerError()
-    assert exp.message == "Server got itself in trouble"
-    assert exp.status_code == 500
-
-
-def test_not_found():
-    exp = NotFound()
-    assert exp.message == "Nothing matches the given URI"
-    assert exp.status_code == 404
-
-
-def test_permission_denied():
-    exp = PermissionDenied()
-    assert exp.message == "Request forbidden -- authorization will not help"
-    assert exp.status_code == 403
-
-
 def test_unsupported_method():
     exp = UnsupportedMethod("GET")
     assert exp.message == "Unsupported method: GET"
     assert exp.status_code == 405
 
 
-def test_unauthorized():
-    exp = Unauthorized()
-    assert exp.message == "No permission -- see authorization schemes"
-    assert exp.status_code == 401
+@pytest.mark.parametrize(
+    "custom_exception",
+    [
+        BadGateway,
+        BadRequestError,
+        Conflict,
+        ExpectationFailed,
+        FailedDependency,
+        GatewayTimeout,
+        Gone,
+        ImATeapot,
+        InsufficientStorage,
+        InvalidResolutionError,
+        LengthRequired,
+        Locked,
+        LoopDetected,
+        MisdirectedRequest,
+        NetworkAuthenticationRequired,
+        NotAcceptable,
+        NotExtended,
+        NotFound,
+        NotImplementedFunctionality,
+        PayloadTooLarge,
+        PaymentRequired,
+        PermissionDenied,
+        PreconditionFailed,
+        PreconditionRequired,
+        ProxyAuthenticationRequired,
+        RangeNotSatisfiable,
+        RequestHeaderFieldsTooLarge,
+        RequestTimeout,
+        ServerError,
+        ServiceUnavailable,
+        TooManyRequests,
+        URITooLong,
+        Unauthorized,
+        UnavailableForLegalReasons,
+        UnprocessableEntity,
+        UnsupportedMediaType,
+        UpgradeRequired,
+        VariantAlsoNegotiates,
+    ],
+)
+def test_custom_exception(custom_exception):
+    exp = custom_exception()
+    try:
+        code, msg = exp.__doc__.split(" - ")
+    except ValueError:
+        code, msg = exp.__doc__.strip(" -"), ""
```

Comment:
I think, more cleaner is to update docstrings (they look more consistent thanks to that too) e.g.:
```python
"""426 - """
#       ^ empty sign
```

Reply:
that was initial way - black removes that

Reply:
That's sad 😢  but we agreed to use `black` we need to accept that too.

Reply:
You can split it and thanks to that you will get an empty string automatically: `exp.__doc__.split(" -")` 😉